report/report.tex

```diff
@@ -37,8 +37,30 @@
 \end{IEEEkeywords}
 
 \section{Introduction}
+Digital door locks are becoming more and more common, and as they become more common, the vulnerabilities they suffer from become more and more dangerous. While digital locks are more convenient by offering a keyless entry and exit, they are susceptible to a slew of different attacks, all intended to derive the static PIN that unlocks the digital lock.
```

Comment:
maybe change to be 'most of which', as there are some attacks such as destructive physical attacks which essentially involve kicking the door down